addon/ng2/models/webpack-build-common.ts

```diff
@@ -73,7 +73,11 @@ export function getWebpackCommonConfig(projectRoot: string, sourceDir: string) {
         filename: 'inline.js',
         sourceMapFilename: 'inline.map'
       }),
-      new CopyWebpackPlugin([{from: path.resolve(projectRoot, './public'), to: path.resolve(projectRoot, './dist')}])
+      new CopyWebpackPlugin([{
+        from: path.resolve(projectRoot, './public'), 
+        to: path.resolve(projectRoot, './dist'),
+        ignore: ['**/*.sass', '**/*.scss', '**/*.styl', '**/*.less']
+      }])
```

Comment:
@jkuri public directory should be copied as is, in full. styles should be part of `src`